### ISSUE TITLE
Supervisord web interface behind proxy with HTTPS scheme

### ISSUE THREAD

**Issue by ulrith** [Supervisor/supervisor#827]
It would be great if we able correctly use Supervisord web interface behind proxy with HTTPS scheme. Currently Supervisord controls return Location header with HTTP scheme (which can be easily redirected to HTTPS). But if it resides inside iframe it's blocked by modern browsers (mixed content error).

Apparent solution seems to give more power to user for control of Location header through configuration settings (url parameter?) or use "X-Forwarded-Proto" or "X-Forwarded-Protocol" header as @fengsi [suggested](https://github.com/Supervisor/supervisor/issues/251#issuecomment-32250649).

**Comment by mnaberez** [comment]
Duplicate of #29, #251.
